sellotape/sellotape_dj/main_app/models.py

```diff
@@ -1,8 +1,32 @@
 from django.db import models
 from django.contrib.auth.models import User
 
+# https://docs.djangoproject.com/en/dev/topics/auth/customizing/#extending-the-existing-user-model
+class Profile(models.Model):
+    # Inherits Attributes from User default instance
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    # User avatar, optional: limit the image instance height and width
+    avatar = models.ImageField()
+    country = models.CharField(max_length=100, default='Israel')
+    
+    class City(models.IntegerChoices):
+        tel_aviv = '1', 'Tel Aviv'
+        hamerkaz = '2', 'HaMerkaz'
+        hadarom = '3', 'HaDarom'
+        hatzafon = '4', 'HaTzafon'
+    
+    city = models.IntegerField(
+        choices=City.choices,
+        default=City.tel_aviv,
+    )   
+
+class UserFollowers(models.Model):
```

Comment:
it has to be more clarified who follows who.
_user_followers_id_ followed by _follower_id_?